targets/atlys_vga.py

```diff
@@ -0,0 +1,241 @@
+# Support for the Digilent Atlys board - digilentinc.com/atlys/
+import struct
```

Comment:
Can you import most of the stuff in this file from the `atlys_base.py` file? (Like what the `atlys_hdmi2usb.py` does?)